Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -84,6 +86,15 @@ def post_decommission(self, resource_id):
         return self._http_request(method='POST', url_suffix=url_suffix, resp_type='response')
 
 
+def test_module(clientObj):
+    response = clientObj.test_apiModule()
+    statusCode = response.status_code
+    if (statusCode == 200):
+        return ("ok")
+    else:
+        raise RuntimeError('Error: ' + str(response.status_code))
```

Comment:
the error should be more indicative, in this case, words are better than numbers (same for all raised errors).
suggest possible solutions like `Check your credentials` or `Check your connection`...

Reply:
We tried to add that but it wasnt reflecting on the test module. So kept it as is.